subsys/logging/Kconfig

```diff
@@ -120,5 +120,130 @@ config SYS_LOG_BACKEND_NET_MAX_BUF_SIZE
 
 endif
 
+config LOG
+	bool
+	prompt "Enable Logger"
+	default y
+	help
+	  Global switch for the logger, when turned off log calls will not be
+	  compiled in.
+
+config LOG_RUNTIME_FILTERING
+	bool
+	prompt "Enable runtime reconfiguration of the logger"
+	depends on LOG
+	default n
+	help
+	  Allow runtime configuration of maximal, independent severity 
+	  level for instance. 
+
+config LOG_DEFAULT_LEVEL
+	int
+	prompt "Default log level"
+	depends on LOG
+	default 3
+	range 0 4
+	help
+	  Sets log level for modules which don't specify it explicitly. When
+	  set to 0 it means log wont be activated for those modules.
+	  Levels are:
+	  0 OFF, do not write by default
+	  1 ERROR, default to only write SYS_LOG_ERR
+	  2 WARNING, default to write SYS_LOG_WRN in addition to previous level
+	  3 INFO, default to write SYS_LOG_INF in addition to previous levels
+	  4 DEBUG, default to write SYS_LOG_DBG in addition to previous levels
```

Comment:
Done

Reply:
Hmm... still looks the same. Did you forget to commit it? :)

This would probably look nice as well (http://docutils.sourceforge.net/docs/user/rst/quickref.html#bullet-lists):

```
	  - 0 OFF, do not write by default
	  - 1 ERROR, default to only write SYS_LOG_ERR
	  - 2 WARNING, default to write SYS_LOG_WRN in addition to previous level
	  - 3 INFO, default to write SYS_LOG_INF in addition to previous levels
	  - 4 DEBUG, default to write SYS_LOG_DBG in addition to previous levels
```